printer_mods/geoffreyyoung/lgx/README.md

```diff
@@ -0,0 +1,66 @@
+# Afterburner + LGX
+
+Useful links, info, and models for using the The LGX™ Large Gears eXtruder with Afterburner...
+
+
+### Afterburner LGX Mounts
+
+Mounting the LXG to AfterBurner requires some additional mounting plates.  You will need both of the following:
+
+  * `lgx_AftB_Mount_Front.STL`
+  * `lgx_AftB_Mount_Rear.STL`
+
+These models are pinned in the [#voronuser_mods](https://discord.com/channels/460117602945990666/635687829254701107/823914498808217621) channel on Discord, and can be found in [this message](https://discord.com/channels/460117602945990666/635687829254701107/823914498808217621).
+
+Thanks to `@Nemgrea V2.199 V0.000` and the crew for their efforts designing and testing the mounts.
+
+For my own build, I found that my third-party M3x20 screws had a large enough head that the toolhead couldn't mount to the X Carriage assembly properly without a bit more clearance.  If you find yourself in this situation this model is nearly identical to the one on Discord but with slightly more clearance for the bottom screws.
+
+  * [`lgx_AftB_Mount_Front.stl`](STLs/lgx_AftB_Mount_Front.stl)
+
+### Klipper config
+
+Bondtech has some official documentation for configuring Klipper [here](https://www.bondtech.se/product-category/upgrade-kits/lgx/).  Below is the configuration I am currently running.
+
+```
+rotation_distance: 8
+microsteps: 16
+full_steps_per_rotation: 200
+
+# use either
+#    gear_ratio: 45:14, 37:17
```

Comment:
The first ratio should be 44:14, not 45:14:
<img width="607" alt="Screen Shot 2021-10-08 at 21 25 11" src="https://user-images.githubusercontent.com/69389593/136639148-89caca89-03f1-4ece-8431-eca0f0449786.png">

Reply:
done.  thanks!